src/routes/CardsPage/CardsPage.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import Container from '@mui/material/Container';
+import { Box, Grid } from '@mui/material';
+import PropTypes from 'prop-types';
+import CardItem from '../../components/card/card-item/CardItem';
+import CardDialog from '../../components/card/card-dialog/CardDialog';
+import { cardApi } from '../../services/apiClient';
+import SideBar from '../../components/sidebar/SideBar';
+
+function CardsPage({ drawerWidth }) {
```

Comment:
![image](https://user-images.githubusercontent.com/24548451/150787311-de68a15e-4e06-4987-8f8b-8912f82e6d56.png)
If package includes only pages why don't you name it pages ?

Reply:
fixed, last naming experience)